website/README.md

```diff
@@ -1,86 +1,92 @@
-
-# Start the workspace
-devpod up tag-app-delivery --ide openvscode
-```
-
-Once the workspace is up, DevPod will start your IDE. Open up a terminal to execute the server startup script
-
-```
-/bin/bash .devcontainer/start-server.sh
-```
-Navigate to http://localhost:1313 with your browser to view the site.
-
-### Run directly on machine
-
-To set up a local dev environment make sure you have [Hugo Extended](https://gohugo.io/installation/linux/#editions) and [npm](https://www.npmjs.com/) installed, then run the server startup script:
-
-```
-/bin/bash .devcontainer/start-server.sh
-```
-the output will tell you which port the webserver is serving.
-
-## Add content to website
-Generally speaking, there are three types of content we feature on the website:
-1. **Projects looking for contributions**  
-These are strictly scoped in both time and content. People can contribute without the need for committing long-term.
-1. **Blogposts**  
-Updates about the TAG.
-1. **Working Groups**  
-Longrunning interest groups, the scope of which might change. A working group can create multiple projects.
+Here's how to create templates for each type of content.
 
 ### Create new call for contribution
+
 ```
 cd website
 hugo new content contribute/<project title>.md
 ```
 
 ### Create new blogpost
+
 ```
 cd website
 AUTHOR=ll hugo new content blog/<post title>.md
 ```
 
 ### Create new working group
+
 ```
 cd website
 hugo new wgs/<working group name>
-```
\ No newline at end of file
+```
+
+## Testing locally
+
+### Run in local OS
+
+To run the site from your local OS:
+
+1. Install [Hugo Extended](https://gohugo.io/installation/linux/#editions) and [npm](https://www.npmjs.com/)
+2. Run `git clone git@github.com:cncf/tag-app-delivery.git && cd tag-app-delivery`
+3. Initialize submodules with `git submodule update --init --recursive`
+4. Change into the website directory: `cd website`
+5. Install dependencies with `npm install`
+6. Run the site using `hugo server -D`
+7. Output from the previous command includes the address to browse to preview the site, by default <http://localhost:1313/>.
+
+Alternatively, once you've cloned this repo you can directly invoke the included
+startup script: `./.devcontainer/start-server.sh`
+
+### Run in a special container
+
+You can create a special "development container", also known as a "workspace",
+that includes all dependencies required to build and run the site. This reduces
+potential differences between local developer environments.
+
+To run the site from a development container:
+
+1. Install [DevPod](https://devpod.sh/docs/getting-started/install)
+1. Install a [DevPod provider](https://devpod.sh/docs/managing-providers/what-are-providers)
+   such as [docker](https://www.docker.com/), using `devpod provider add docker`.
+
+### GUI
+
+Open the environment directly:
+
+[![Open in DevPod!](https://devpod.sh/assets/open-in-devpod.svg)](https://devpod.sh/open#https://github.com/cncf/tag-app-delivery)
+
+### CLI
```

Comment:
⚠️ untested nit 😬 
The changes in here are two fold:
1. Reference more detail about the start-server script if someone decides to go that way
2. Nest the GUI and CLI headers under the other GitPod header since my understanding is that those are subsections

This is *NOT* blocking, but I think makes it easier to parts 🤞 

```suggestion
Alternatively, once you've cloned this repo you can directly invoke the included
startup script: `./.devcontainer/start-server.sh` as described in the [special
container CLI](#cli) section.

### Run in a special container

You can create a special "development container", also known as a "workspace",
that includes all dependencies required to build and run the site. This reduces
potential differences between local developer environments.

To run the site from a development container:

1. Install [DevPod](https://devpod.sh/docs/getting-started/install)
1. Install a [DevPod provider](https://devpod.sh/docs/managing-providers/what-are-providers)
   such as [docker](https://www.docker.com/), using `devpod provider add docker`.

#### GUI

Open the environment directly:

[![Open in DevPod!](https://devpod.sh/assets/open-in-devpod.svg)](https://devpod.sh/open#https://github.com/cncf/tag-app-delivery)

#### CLI
```

Reply:
Addressed these in my commits